plumbing/format/packfile/parser.go

```diff
@@ -378,6 +424,9 @@ func (p *Parser) get(o *objectInfo, buf *bytes.Buffer) (err error) {
 		}
 	}
 
+	// If the scanner is seekable, caching this data into
+	// memory by offset seems wasteful.
+	// !p.scanner.IsSeekable &&
```

Comment:
Is this a `TODO`?

Reply:
@hiddeco well spotted, thanks for that. In between performance tests I removed that as the further improvements on memory consumption had an impact on execution time. Added the `// TODO` and some additional comments for future reference. PTAL